about/docdownload.Rmd

```diff
@@ -19,40 +19,40 @@ Each measure used in the MWI Tool was assessed based on its strengths, limitatio
-- [Suicide Mortality](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Suicide_Mortality.docx?raw=true){target="_blank"}
-- [Life Expectancy](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Life_Expectancy.docx?raw=true){target="_blank"}
-- [Suicidal Ideation](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Suicidal_Ideation.docx?raw=true){target="_blank"}
+- [Adult Binge Drinking](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Adult_Binge_Drinking.pdf?raw=true){target="_blank"}
+- [Insufficient Sleep](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Insufficient_Sleep.pdf?raw=true){target="_blank"}
+- [Obesity](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Obesity.pdf?raw=true){target="_blank"}
+- [Poor Mental Health](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Poor_Mental_Health.pdf?raw=true){target="_blank"}
+- [Smoking Status](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Smoking_Status.pdf?raw=true){target="_blank"}
+- [Alcohol-Related Mortality](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Alcohol_Related_Mortality.pdf?raw=true){target="_blank"}
+- [Estimated Drug Poisoning Mortality](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Estimated_Drug_Poisoning_Mortality.pdf?raw=true){target="_blank"}
+- [Suicide Mortality](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Suicide_Mortality.pdf?raw=true){target="_blank"}
+- [Life Expectancy](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Life_Expectancy.pdf?raw=true){target="_blank"}
+- [Suicidal Ideation](https://github.com/mitre/hse-mwi/blob/main/Documentation/Measures/Suicidal_Ideation.pdf?raw=true){target="_blank"}
 
 # Methods
-- [Facility Weights](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Facility%20Weights%20Methods.docx?raw=true){target="_blank"}
-- [Geography](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Geography%20Methods.docx?raw=true){target="_blank"}
-- [Two Step Floating Catchment Methodology](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Two%20Step%20FCA.docx?raw=true){target="_blank"}
-- [Facility Weighting](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Weighting%20Methods.docx?raw=true){target="_blank"}
\ No newline at end of file
+- [Facility Weights](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Facility%20Weights%20Methods.pdf?raw=true){target="_blank"}
+- [Geography](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Geography%20Methods.pdf?raw=true){target="_blank"}
+- [Two Step Floating Catchment Methodology](https://github.com/mitre/hse-mwi/blob/main/Documentation/Methods/Two%20Step%20FCA.pdf?raw=true){target="_blank"}
```

Comment:
FYI The file name for the two step FCA document has changed from the acronym to the full words :)

Reply:
Fixed!